bin/build

```
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+git fetch --tags  # jenkins does not do this automatically yet
+
+docker-compose pull goreleaser
+
+echo "> Building and packaging binaries"
+docker-compose run --rm \
+  -T \
+  --entrypoint sh \
+  goreleaser -es << EOF
+    goreleaser release --rm-dist --skip-validate
+EOF
```

We can avoid wrapping with sh without losing any functionality.

```suggestion
docker-compose run --rm \
  -T \
  goreleaser release --rm-dist --skip-validate
```